scripts/clang-format.py

```diff
@@ -14,7 +14,7 @@
     'lib',
     'master',
 ]
-husky_root = os.getenv('HUSKY_ROOT', '.')
+husky_root = '.';
```

Comment:
No. If so, you are limiting user to run this script in `husky/`. If they want to run the script in another path, then they need to set the `HUSKY_ROOT` at first. Get it?